test/CMakeLists.txt

```diff
@@ -14,6 +14,36 @@
 # limitations under the License.
 # ******************************************************************************
 
+if(NGRAPH_UNIT_TEST_NUMPY_ENABLE)
+    include(ExternalProject)
+
+    ExternalProject_Add(
+        ext_pybind11
+        PREFIX ut_pybind11
+        GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
+        GIT_TAG "v2.5.0"
```

Comment:
Is there a way to make sure the TAG here and in python/CMakeLists.txt is same?

Reply:
Yes there is, defined it in the top-level cmake file and then reference it in the two places